docs/learn-launch.md

```diff
@@ -70,3 +70,9 @@ parachain slot auctions, parathreads, and cross-chain message passing. These upg
 runtime upgrades that will pass through Polkadot's normal governance processes. The core
 functionality does not have to be unlocked sequentially &mdash; several features can be unlocked
 with a single proposal.
+
+## Polkadot 2.0
+
+After the 1.0 version of Polkadot is released, research for the next version of Polkadot will
```

Comment:
Version 1.0 is _technically_ released, and 2.0 research is _technically_ underway. I don't think this section adds much value as it tbh.